### PR TITLE
Feature/temp files windows

### PR DESCRIPTION
**Pull request by holtgrewe**
Also, bumping Mason version to 2.0.5.

### CODE REVIEW

File: apps/mason2/external_split_merge.cpp [hunk 1]
```diff
@@ -40,56 +40,60 @@
...
+		mode_t cur_umask = umask(S_IRWXO | S_IRWXG);  // to silence Coverity warning
+		int fd = mkstemp(&pathTpl[0]);
+		files.push_back(new std::fstream(pathTpl.c_str(), std::ios::binary | std::ios::in | std::ios::out));
+		umask(cur_umask);
+		remove(pathTpl.c_str());
+		::close(fd);
 #endif
 
-        if (!files.back())
+		if (!files.back() || !files.back()->good())
```
Comment (rrahn) [inline apps/mason2/external_split_merge.cpp]: broken indentation?

File: apps/mason2/CMakeLists.txt [hunk 2]
```diff
@@ -26,10 +26,10 @@ if (NOT ZLIB_FOUND)
   return ()
 endif (NOT ZLIB_FOUND)
 
-if (MSVC)
-  message (STATUS "  Not building MASON2 under Visual Studio.")
-  return ()
-endif()
+# Mason2 does not work correctly with Visual Studio Version <10 on 64 bit systems.
+if (MSVC AND MSVC_VERSION LESS 1600)
```
Comment (rrahn) [inline apps/mason2/CMakeLists.txt]: I think you won't need this.
For the entire SeqAn repo we require at least VS15 to get full c++11 support.

**Comment by rrahn** [comment]
@holtgrewe still some open issues. I'll postpone it to 2.3.0

**Comment by h-2** [comment]
@holtgrewe Please update this PR. If it is not merged before 2.3.0 it will be closed. Thank you!

**Pull request closed without merging**